backend/medtagger/api/users/service.py

```diff
@@ -57,6 +57,23 @@ def get() -> Any:
         return user, 200
 
 
+@users_ns.route('/<int:user_id>/settings')
+class SetUserSettings(Resource):
+    """Set user's settings."""
+
+    @staticmethod
+    @login_required
+    @users_ns.doc(security='token')
+    def post(user_id: int) -> Any:
+        """Set current user's settings."""
+        user = get_current_user()
+        if user.id != user_id:
+            return {}, 403
```

Comment:
Please add and raise an Exception for Access Forbidden to `exceptions.py` and add its handler in `api/__init__.py`. Pass some details why his access is forbidden - something like "Cannot update Settings for someone else."